changelog.md

```diff
@@ -1,19 +1,23 @@
 # Changelog
 
-## v0.6.0 | t.b.d.
+## v0.6.0 | 2020-08-18
+
+* Plot and return images and timestamps for scans using physical coordinate system rather than fast and slow scanning axis. See docs tutorial section on `Confocal images` and `Kymographs` on how to use the API. **Note that this is a breaking change for scans with the fast axis in y direction!**
+* `Scan.pixels_per_line` and `Kymo.pixels_per_line` return the number of pixels along the fast axis now. **Note that this is a breaking change for scans/kymos with the fast axis in y direction!**
```

Comment:
I feel like this should be rolled into the previous point.

Reply:
Done.